styles/gmail/catppuccin.user.css

```diff
@@ -876,6 +876,19 @@
         box-shadow: inset 0 -2px 0 0 @accent-color;
       }
     }
+    /* Empty social/promotions tabs */
+    .aRs {
+        background-color: @base;
+        color: @subtext0;
+    }
+      
+    .aRv {
+        color: @text;
+    }
+      
+    .aRq {
+        color: @accent-color;
+    }
```

Comment:
Let's put this in/after the 578-613 section where other styling for stuff in the same visual location. Also, don't separate these declaration blocks by double newlines - that is only for unrelated or nested CSS. This is all the same area so just one newline / no gap.